admin/tabs/general/roles/graph/RoleGraphModel.ts

```diff
@@ -29,23 +31,85 @@ export class RoleGraphModel extends HoistModel {
         return this.roleModel.selectedRole;
     }
 
+    get leafCount(): number {
```

Comment:
Nit - I'm a fan of the `lodash.sumBy` / tend to prefer that over `reduce` for readability. Also might be worth decorating with `@computed` .

Reply:
Another nit - could this be made `private`?